src/containers/BooksForm.js

```diff
@@ -1,18 +1,49 @@
 import React from 'react';
-
+import { createBook }  from '../actions/index';
+import { connect } from 'react-redux';
+ 
 const bookCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
 
-const BooksForm = () => (
-  <form>
-    <label htmlFor="title-input">Title</label>
-    <input id="title-input" type="text" />
-    <select>
-      {bookCategories.map((category, index) => (
-        <option value="category" key={index}>{category}</option>
-      ))}
-    </select>
-    <input type="submit" />
-  </form>
-);
+class BooksForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+          title: '',
+          category: '',
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+    
+    handleChange(event) {        
+        this.setState({ [event.target.id]: event.target.value });                
+    }
+    handleSubmit(event){
+        event.preventDefault();
```

Comment:
This file is not properly formatted, the indentation and spacing between methods, not correct. 
Please fix